src/cardsVersion2/TestCards.java

```diff
@@ -0,0 +1,49 @@
+package cardsVersion2;
+
+import java.util.Map;
+import java.util.Scanner;
+
+public class TestCards {
+    public static void main(String[] args) {
+
+        System.out.println("Podaj jezyk PL / EN");
+        Scanner scanner = new Scanner(System.in);
+        String userChoice = scanner.nextLine();
+
+        Card card = new Card();
+        Map<String, String> setTranslated = card.translate(userChoice);
```

Comment:
po tym jak stworzysz 52 karty mozesz wyswietlic ich tłumaczenie. Mozesz to w łatwy sposób zrobić poprzez dodanie metod : `getPolishDescription` i `getEnglishDescription` do klasy `Card`

Reply:
poprawione